### ISSUE TITLE
Intelligent merging of adjacent subtitles when recording audio

### ISSUE THREAD

**Issue by killergerbah** [killergerbah/asbplayer#396]
Sometimes a single sentence might be split across multiple lines and it would be nice if asbplayer could reliably detect this case during audio recording so that a recording does not appear to stop mid-sentence.

**Comment by killergerbah** [comment]
@LeonelRFF How is this different from the slider at the bottom of the Anki dialog? 
![image](https://github.com/user-attachments/assets/0e0fd9b3-d743-4c1a-9361-6013fbe553ea)

**Comment by LeonelRFF** [comment]
Thanks for replying, yes, I use the slider, but trimming audio only works with local videos, not with online videos, it only adds or removes text from the subtitles, I don't see that it trims the audio even though it has been recorded, could this be addressed in a separate issue? since the idea is that it works with **the audio fragment recorded at the time of mining**
I have seen a similar topic to this #560 but in that case it was local videos, my suggestion is for online 😊

**Comment by killergerbah** [comment]
@LeonelRFF Yes it will work with audio if you rerecord it using the big circle button in the audio field. Its not super clear from the UI though and that can be improved.
